collector/nodes.go

```diff
@@ -286,6 +286,66 @@ func NewNodes(logger *slog.Logger, client *http.Client, url *url.URL, all bool,
 				},
 				Labels: defaultNodeLabelValues,
 			},
+			{
```

Comment:
I think these metrics should be converted to seconds. It's typical practice for prometheus metrics to always be in base units - https://prometheus.io/docs/practices/naming/#metric-names